src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -1025,12 +1022,12 @@ namespace WindowEquivalentLayer {
         QAllSWwinAbs({1, NL + 1}) = QRadSWwinAbs({1, NL + 1}, SurfNum);
         //  Solve energy balance(s) for temperature at each node/layer and
         //  heat flux, including components, between each pair of nodes/layers
-        ASHWAT_ThermalR = ASHWAT_Thermal(
-            CFS(EQLNum), TIN, Tout, HcIn, HcOut, TRMOUT, TRMIN, 0.0, QAllSWwinAbs({1, NL + 1}), TOL, QOCF, QOCFRoom, T, Q, JF, JB, H, UCG, SHGC);
-        // long wave radiant power to room not including reflected
-        QRLWX = JB(NL) - (1.0 - LWAbsIn) * JF(NL + 1);
-        // nominal surface temp = effective radiant temperature
-        SurfInsideTemp = TRadC(QRLWX, LWAbsIn);
+        ASHWAT_ThermalR =
+            ASHWAT_Thermal(CFS(EQLNum), TIN, Tout, HcIn, HcOut, TRMOUT, TRMIN, 0.0, QAllSWwinAbs({1, NL + 1}), TOL, QOCF, QOCFRoom, T, Q, JF, JB, H);
+
+        // effective surface temperature is set to surface temperature calculated
+        // by the fenestration layers temperature solver
+        SurfInsideTemp = T(NL) - KelvinConv;
```

Comment:
So QRLWX is just not set anymore?  I need to see exactly what TRadC was doing.

Reply:
QRLWX was used to re-calculate the inside surface temperature. I was hesitant about the use this variable when I first implemented the model. The inside surface temperature is already calculated using layer by layer thermal balance function ASHWAT_Thermal() and has average room radiant boundary temperature as an input to this function. I always thought that QRLWX calculation was unnecessary step. It is the re-calculation of the inside temperature using QRLWX that resulted in unrealistic inside surface temperature and lead to simultaneous heating and cooling situation for this defect. That is why I removed it.